### PR TITLE
Done

### PR DESCRIPTION
**Pull request by otalu**

### CODE REVIEW

File: capitals.py [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+is and then gives three random capitals along with the right one in a
+randomized order.
+Every correct answer is 10 and every false answer is -10 points. Game doesn't
+continue unless each question is answered correctly.
+It takes the capital-country information from a dictionary in the
+script countries.py.
+
+Authors = Anupama Krishnan, Onur Talu
+"""
+
```
Comment (SungwooPark) [inline capitals.py:12]: Good header comment.

File: capitals.py [hunk 2]
```diff
@@ -0,0 +1,129 @@
+"""
+Makes a 10 questions long trivia game that asks what the capital of a country
+is and then gives three random capitals along with the right one in a
+randomized order.
+Every correct answer is 10 and every false answer is -10 points. Game doesn't
```
Comment (SungwooPark) [inline capitals.py:5]: When I run the program, wrong answer does not deduct the points.

Comment (SungwooPark) [reply capitals.py:5]: Nevermind, I was confused because the score did not update instantly. Is there any way to update those scores right away?

**Comment by otalu** [comment]
The program doesn't let you move on until you get it right.
If you answer wrong you get -10, the second time you get out right you get
+10. So overall it's neutral.
If you get it won't twice before getting it right, you end up with -10 etc.

**Comment by SungwooPark** [comment]
Solid implementation with good readability and appropriate amount of documentation. One thing that I would suggest is maybe try using classes to organize your program? If you do use classes, there is no need for global variables, which is usually not a good idea. Good work!

**Comment by otalu** [comment]
There is, but as far as I remember not in the way we structured the code.
It's still doable though

**Pull request closed without merging**